src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/ComAwareEventInfo.cs

```diff
@@ -2,6 +2,24 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Runtime.InteropServices;
+namespace System.Runtime.CompilerServices
+{
+    [Serializable]
+    [System.AttributeUsage(AttributeTargets.Field | AttributeTargets.Parameter, Inherited=false)]
+    [System.Runtime.InteropServices.ComVisible(true)] 
+    public sealed partial class IUnknownConstantAttribute : System.Runtime.CompilerServices.CustomConstantAttribute 
+    {
+        public IUnknownConstantAttribute() { }
+        public override object Value
+        { 
+            get 
+            { 
+                return new UnknownWrapper(null); 
+            } 
+        }
+    }
+}
```

Comment:
Why is this type being added to ComAwareEventInfo.cs?  Shouldn't it be added to its own file?  It's not even in the same namespace :)